group_name: "manipulator"
base_frame: "world"
tool_frame: "flange"
poi_names: ["a_forearm","b_mount_flange","b_forearm","flange"]

n_objects: 1
min_range: [-2,-2,0]
max_range: [ 2, 2,1.5]
max_distance: 2.5
norm_1: true

n_iter: 500000
database_name: "ssm_dataset_distance"
